universal-login-react/src/ui/TopUp/Fiat/FiatFooter.tsx

```diff
@@ -53,7 +53,6 @@ export const FiatFooter = ({paymentMethod}: FiatFooterProps) => {
     case TopUpProvider.WYRE:
       return (
         <div className="info-block info-row">
-          <p className="info-text info-text-hint">Available soon</p>
         </div>
```

Comment:
#nextPR 

we can add icons here, for example, Apple pay icon

Reply:
👍